types/difflib/difflib-tests.ts

```diff
@@ -1,7 +1,23 @@
 import { Differ, SequenceMatcher, contextDiff, unifiedDiff } from 'difflib';
 
 const sequenceMatcher = new SequenceMatcher(null, ['first', 'second'], ['third', 'fourth']);
+// $ExpectType (char: string) => boolean | undefined
+const setSeqs = sequenceMatcher.setSeqs(['five', 'six'], ['seven', 'eight']);
+// $ExpectType null | undefined
+const setSeq1 = sequenceMatcher.setSeq1(['nine', 'ten']);
+// $ExpectType (char: string) => boolean | undefined
```

Comment:
I think it is `((char: string) => boolean) | undefined` instead of `(char: string) => boolean | undefined`, don't you think?
[This line](https://github.com/qiao/difflib.js/blob/e11553ba3e303e2db206d04c95f8e51c5692ca28/lib/difflib.js#L284) returns an undefined, whereas the type provided implies a function that return either a boolean or undefined...